internal/cluster/validator.go

```diff
@@ -400,6 +401,27 @@ func (v *clusterValidator) allHostsAreReadyToInstall(c *clusterPreprocessContext
 	return ValidationFailure, "The cluster has hosts that are not ready to install."
 }
 
+func (v *clusterValidator) platformRequirementsSatisfied(c *clusterPreprocessContext) (ValidationStatus, string) {
+	// If cluster platform type is not OCI ignore that validation
+	if !(c.cluster.Platform != nil && common.PlatformTypeValue(c.cluster.Platform.Type) == models.PlatformTypeOci) {
+		return ValidationSuccess, "Platform requirements satisfied"
+	}
+
+	usages, err := usage.Unmarshal(c.cluster.Cluster.FeatureUsage)
```

Comment:
why we are using the feature usage for checking it and not data from the cluster? what will happen if there will be a bug in the feature usage?